docs/src/pages/components/app-bar/ResponsiveAppBar.js

```diff
@@ -0,0 +1,238 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import MenuItem from '@mui/material/MenuItem';
+import Menu from '@mui/material/Menu';
+import MenuIcon from '@mui/icons-material/Menu';
+import Container from '@mui/material/Container';
+import Avatar from '@mui/material/Avatar';
+
+const ResponsiveAppBar = () => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [anchorElp, setAnchorElp] = React.useState(null);
+
+  const handleMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleMenup = (event) => {
+    setAnchorElp(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleClosep = () => {
+    setAnchorElp(null);
+  };
+
+  return (
+    <AppBar position="static">
+      <Container maxWidth="xl">
+        <Toolbar>
+          <Typography
+            variant="h6"
+            noWrap
+            component="div"
+            sx={{ display: { xs: 'none', md: 'flex' } }}
+          >
+            LOGO
+          </Typography>
+
+          <Box sx={{ flexGrow: 1, display: { xs: 'flex', md: 'none' } }}>
+            <IconButton
+              size="large"
+              aria-label="account of current user"
+              aria-controls="menu-appbar"
+              aria-haspopup="true"
+              onClick={handleMenu}
+              color="inherit"
+            >
+              <MenuIcon />
+            </IconButton>
+            <Menu
+              id="menu-appbar"
+              anchorEl={anchorEl}
+              anchorOrigin={{
+                vertical: 'top',
+                horizontal: 'right',
+              }}
+              keepMounted
+              transformOrigin={{
+                vertical: 'top',
+                horizontal: 'right',
+              }}
+              open={Boolean(anchorEl)}
+              onClose={handleClose}
+              sx={{
+                display: { xs: 'block', sm: 'none' },
+                mt: '32px',
+                px: '20px',
+              }}
+            >
+              <MenuItem>
```

Comment:
```suggestion
              <Button>
```
Currently does not work with `Tab` keyboard.

Reply:
why need to change  <MenuItem> to  <Button> ? <MenuItem> still working well.  @siriwatknp @siriwatknp

Reply:
Here, I thought you suggest me to remove <MenuItem> here. that's why I remove <MenuItem>. I'm going to add <MenuItem> again.